app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -100,7 +100,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           :data => {'toggle'        => 'modal',
                     'target'        => '#modal_d_div',
                     'function'      => 'miqCallAngular',
-                    'function-data' => '{"name": "showListener", "args": []}'})
+                    'function-data' => '{\"name\": \"showListener\", \"args\": []}'})
```

Comment:
Aaah, didn't knew there is such function. I've changed it back to original value and used `j_str` and it works same as with escaped string.